### PR TITLE
Ashwani forking. Just a test for forking.

### PR DESCRIPTION
**Pull request by AV-013**

### CODE REVIEW

**Review by pradipta — CHANGES_REQUESTED** [review]
Lol.
In the latest blog there's a line: `To use the same with a Java, you would need the following dependencies (in addition to the Mongo Java Driver/Spring Mongo data that you’d use to set up the application anyways):`. Change it to `To use the same with Java.....`. Remove the `a`

**Comment by AV-013** [comment]
> Lol.
> In the latest blog there's a line: `To use the same with a Java, you would need the following dependencies (in addition to the Mongo Java Driver/Spring Mongo data that you’d use to set up the application anyways):`. Change it to `To use the same with Java.....`. Remove the `a`

I dont think i can edit your blog ?

**Comment by AV-013** [comment]
check this

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Jekyll based Github Page
\ No newline at end of file
+Jekyll based Github Page
+
+Ashwani forking example. Just a test .......
```
Comment (pradipta) [inline README.md:3]: Revert this change

**Review by pradipta — CHANGES_REQUESTED** [review]

**Review by pradipta — APPROVED** [review]

**Pull request merged**